src/main/java/com/box/sdk/BoxStoragePolicyAssignment.java

```diff
@@ -0,0 +1,273 @@
+package com.box.sdk;
+
+import java.net.URL;
+
+import com.box.sdk.http.HttpMethod;
+
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+import com.eclipsesource.json.ParseException;
+
+/**
+ * Represents a BoxStoragePolicyAssignment.
+ */
+@BoxResourceType("BoxStoragePolicyAssignment")
```

Comment:
I think the string in this annotation is supposed to be the Box type, e.g. `"storage_policy_assignment"`